src/main/java/teammates/ui/controller/ActionFactory.java

```diff
@@ -88,6 +88,7 @@
         map(INSTRUCTOR_FEEDBACK_QUESTION_COPY_PAGE, InstructorFeedbackQuestionCopyPageAction.class);
         map(INSTRUCTOR_FEEDBACK_QUESTION_COPY, InstructorFeedbackQuestionCopyAction.class);
         map(INSTRUCTOR_FEEDBACK_QUESTION_EDIT, InstructorFeedbackQuestionEditAction.class);
+        map(INSTRUCTOR_FEEDBACK_QUESTION_RESULTS_DOWNLOAD, InstructorFeedbackQuestionResultsDownloadAction.class);
```

Comment:
Just a discussion:  INSTRUCTOR_FEEDBACK_QUESTION_RESULTS_DOWNLOAD to me looks as if it is downloading different thing from INSTRUCTOR_FEEDBACK_RESULTS_DOWNLOAD, but actually they are doing the same thing but just with one more attribute. Wondering if you should just modify InstructorFeedbackQuestionResultsDownloadAction instead of creating a new action class